views/home/index.ejs

```diff
@@ -1,16 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="shortcut icon" href="#" />
+  </head>
 
 <h1 class="title">Racket</h1>
 <h3>Welcome to Racket</h3>
 
-<p>Don't have an account? Sign up now!</p>
-<form id="new-user-form" action="/users" method="post">
-  <label for="email">Email: </label>
```

Comment:
see [input tag for email](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email)

for some form validation, include required for all inputs too please